lib/docker/connection.rb

```diff
@@ -38,17 +38,17 @@ def request(*args, &block)
     request = compile_request_params(*args, &block)
     log_request(request)
     resource.request(request).body
-  rescue Excon::Errors::BadRequest => ex
+  rescue Excon::Error::BadRequest => ex
     raise ClientError, ex.response.body
-  rescue Excon::Errors::Unauthorized => ex
+  rescue Excon::Error::Unauthorized => ex
     raise UnauthorizedError, ex.response.body
-  rescue Excon::Errors::NotFound => ex
+  rescue Excon::Error::NotFound => ex
     raise NotFoundError, ex.response.body
-  rescue Excon::Errors::Conflict => ex
+  rescue Excon::Error::Conflict => ex
     raise ConflictError, ex.response.body
-  rescue Excon::Errors::InternalServerError => ex
+  rescue Excon::Error::InternalServerError => ex
```

Comment:
I'm actually not sure about this:

https://github.com/excon/excon/commit/d2a550c052b0e904d247d85314907bf366213c8e